.github/workflows/unit-tests.yml

```
@@ -0,0 +1,34 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

```suggestion
    strategy:
      fail-fast: false
      matrix:
        python-version: [3.7, 3.8]
```

Create matrix of different python versions